hackathon_site/event/serializers.py

```diff
@@ -86,6 +86,22 @@ def update(self, instance: Profile, validated_data):
         return serializers.ModelSerializer.update(self, instance, validated_data)
 
 
+class CreateProfileSerializer(ProfileSerializer):
+    class Meta(ProfileSerializer.Meta):
+        read_only_fields = ("id", "team", "id_provided", "attended")
+
+    def create(self, validated_data):
+        response_data = {"attended": True}
```

Comment:
you need to do some validation here with the user

if `self.context["request"].user.profile" exists then throw a ValidationError with a "user already has profile" message